src/glog/logging.h.in

```diff
@@ -1327,7 +1327,7 @@ inline void LogAtLevel(int const severity, std::string const &msg) {
 //     : x_(CHECK_NOTNULL(MethodReturningUniquePtr())) {}
 template <typename T>
 T CheckNotNull(const char* file, int line, const char* names, T&& t) {
```

Comment:
Don't you need to remove the r-value reference (`T&& t`) as well in order to support C++98/03?

Reply:
Actually, the usage of C++11 features is already guarded by defines. No changes should be needed.